src/coreclr/jit/llvm_types.h

```diff
@@ -0,0 +1,83 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*
+ * Types used in LLVM compilation that are abstractions of the VM types
+*/
+
+#ifndef _LLVM_TYPES_H_
+#define _LLVM_TYPES_H_
+
+#include "corinfo.h"
```

Comment:
It is unusual to directly include `corinfo.h` in Jit code. Would including `jitpch.h` not work?